ui-core/src/styles/inputs/fieldWrapper.css

```
@@ -28,5 +32,53 @@
         margin-left: 0.563rem;
       }
     }
+
+    input:focus:has(.status-message-wrapper-tooltip) .status-message-wrapper-tooltip {
+      z-index: 2;
+    }
+
+    .status-message-wrapper-tooltip {
+      position: absolute;
+      display: flex;
+      align-items: center;
+      width: 320px;
+      top: 60px;
```

To follow easier the mockup behavior (4px offset), maybe we can use this (the tooltip is too high atm)
```suggestion
      top: calc(100% - inputMargin - fieldMargin - 4px);
```